packages/react-composites/src/composites/ChatComposite/FileDownloadCards.tsx

```diff
@@ -54,6 +58,9 @@ export const FileDownloadCards = (props: FileDownloadCards): JSX.Element => {
           setShowSpinner(false);
           if (response instanceof URL) {
             window.open(response.toString(), '_blank', 'noopener,noreferrer');
+          } else {
+            console.log(response);
```

Comment:
No need to log it to console here.
Contoso returns this error so they already know about it.

Reply:
Done